audformat/core/testing.py

```diff
@@ -217,6 +217,10 @@ def create_db(minimal: bool = False) -> Database:
         return db
 
     db.description = 'A database for unit testing.'
+    db.license = 'CC0'
```

Comment:
Is there a specific reason you don't want to use one of our pre-defined licenses and let it fill in `license_url` automatically?

Reply:
No, I just forgot to change this.
Fixed it now.